k8s/prod/common/financial-remedy/finrem-cos.yaml

```diff
@@ -25,6 +25,12 @@ spec:
         singleRequestTcp: false
       environment:
         IDAM_API_URL: https://idam-api.platform.hmcts.net
+        FEATURE_APPROVED_CONSENT_ORDER_NOTIFICATION_LETTER: false
+        FEATURE_HWF_SUCCESSFUL_NOTIFICATION_LETTER: false
+        FEATURE_ASSIGNED_TO_JUDGE_NOTIFICATION_LETTER: false
+        FEATURE_CONSENT_ORDER_NOT_APPROVED_APPLICANT_DOCUMENT_GENERATION: false
+        BSP_SERVICES_ALLOWED_TO_UPDATE:
```

Comment:
ok, but it means it will not be available on prod

Reply:
Yep that's the plan - I'll create a separate PR that will be merged on Monday that will actually enable it for Prod